releasenotes/notes/slice-bug-1aa7af1b87d4a9df.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed a bug in :func:`qiskit_addon_obp.backpropagate` which caused slices with depth > 1 to be incorrectly forward-propagated.
```

Comment:
Can we be more explicit and state that gates within a slice will from now on be backpropagated properly, that is, in reverse order.